apps/src/javalab/Javalab.js

```diff
@@ -90,6 +90,15 @@ Javalab.prototype.init = function(config) {
     visualization = <DefaultVisualization />;
   }
 
+  Javalab.handleVersionHistory = this.studioApp_.getVersionHistoryHandler(
+    config
+  );
+
+  config.afterClearPuzzle = function() {
```

Comment:
This should be defined outside the Javalab.init. Similar to how we handle getCode.

Reply:
(And should be moved up by `getCode`

Reply:
Oh gotcha- I'll do this. Thanks!